sqlwrapper/ghostferry_db.go

```diff
@@ -86,3 +88,7 @@ func (tx Tx) QueryRowContext(ctx context.Context, query string, args ...interfac
 func (tx Tx) QueryRow(query string, args ...interface{}) *sqlorig.Row {
 	return tx.Tx.QueryRow(query, args...)
 }
+
+func Annotate(query, marginalia string) string {
```

Comment:
does this need to be exported? I think it can be only used internally if we don't need to call it from the test below

Reply:
it's used in race_conditions_integration_test.go to annotated test queries